misc/packaging/apache/ansible-runner-service.spec

```diff
@@ -72,8 +72,8 @@ mkdir -p %{buildroot}%{_var}/www/runnner
 install -m 644 ./wsgi.py %{buildroot}%{_var}/www/runnner/runner.wsgi
 
 %post
-semanage fcontext -a -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log
-[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && restorecon -rF /var/log/ovirt-engine/ansible-runner-service.log
+semanage fcontext -a -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log || true
```

Comment:
Shouldn't either addition or modification return success? Why do we need addition ```|| true```?

Reply:
If we add it once it passes but if it already in the database it fails.
```
[root@10-37-138-22 ~]# semanage fcontext -a -t httpd_log_t -s system_u /var/log/ovirt-engine/ansible-runner-service.log
ValueError: File context for /var/log/ovirt-engine/ansible-runner-service.log already defined
[root@10-37-138-22 ~]# echo $?
1
```

Reply:
Ah, OK, the problem is in the return code of ```[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]]```